rust/otap-dataflow/crates/otap/src/otap_exporter.rs

```diff
@@ -109,6 +112,12 @@ impl local::Exporter<OtapPdata> for OTAPExporter {
                 self.config.grpc_endpoint
             ))
             .await;
+
+        let exporter_id = effect_handler.exporter_id();
+        let _ = effect_handler
```

Comment:
Should we get hold of the `TelemetryTimerCancelHandle` returned here and use it to cancel telemetry timer when processing `NodeControlMsg::Shutdown`?

Reply:
Sounds good. Fixed